include/stb/stb_image.h

```diff
@@ -971,7 +971,8 @@ STBIDEF void stbi_set_flip_vertically_on_load(int flag_true_if_should_flip)
 
 static void *stbi__load_main(stbi__context *s, int *x, int *y, int *comp, int req_comp, stbi__result_info *ri, int bpc)
 {
-   memset(ri, 0, sizeof(*ri)); // make sure it's initialized if we add new fields
+   bpc = 0;
+   memset(ri, bpc, sizeof(*ri)); // make sure it's initialized if we add new fields
```

Comment:
What's the purpose of doing this? You are also overwriting the argument that might be passed to this function. And the same question to the `memset(..., req_comp)` below. Not to mention this is sort of an external dependency, editing it is I guess in bad taste, but whatever.

Reply:
perhaps adding an ifdef to check and see if the image isn't a PSD file, and if so, using the unused define this particular file uses, would fix this

Reply:
What warning is being fixed here? Unused parameter `bpc`?
Might want to just update stbi to a newer version, because there is [this](https://github.com/nothings/stb/blob/master/stb_image.h#L1071).

Reply:
Updated to the latest version already. Resolved.